lib/std/heap.zig

```diff
@@ -578,6 +578,14 @@ pub fn StackFallbackAllocator(comptime size: usize) type {
     };
 }
 
+/// Runtime known minimum page size.
```

Comment:
The fact that it is runtime-known is redundant with the type signature of the function.

It's not really minimum, is it? It's the actual page size.

I think this doc comment should be deleted.

Reply:
Alright, deleted the doc comment.

Reply:
> It's not really minimum, is it? It's the actual page size.

There's more than one page size on many systems: e.g. on x86_64, you can have both a 4KB page and a 2MB page within the one process.